presto-native-execution/presto_cpp/main/operators/tests/PlanNodeSerdeTest.cpp

```diff
@@ -104,7 +40,9 @@ class PlanNodeSerdeTest : public testing::Test,
     Type::registerSerDe();
     core::PlanNode::registerSerDe();
     core::ITypedExpr::registerSerDe();
-    exec::registerPartitionFunctionSerDe();
+    auto& registry = DeserializationWithContextRegistryForSharedPtr();
```

Comment:
#include "velox/exec/PartitionFunction.h"

exec::registerPartitionFunctionSerDe();

Reply:
Thank you for reminding, Ke! Doing the change all over again.